samples/tab-approval/nodejs/Readme.md

```diff
@@ -0,0 +1,155 @@
+---
+page_type: sample
+description: Microsoft Teams sample app for Sending Approval request using activity feed notification.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "06-10-2021 01:48:56"
+---
+
+# Tab request approval with activity feed notification.
+
+This sample shows a feature where:
+1)Requester: Can request for any task to the manager by sending activity feed notification and can see his request status.
+2)Manager : Can see the list of pending request with him on the click of activity feed notification and can approve or reject the request.
```

Comment:
Can see the pending approval request raised by user on the click of activity feed notification and can approve or reject the request.